t/ui/10-tests_overview.t

```diff
@@ -543,18 +543,35 @@ subtest 'filter by result and state' => sub {
     ok !$driver->find_element_by_id('filter-failed')->is_selected, 'other checkbox not checked';
 };
 
+
+subtest "job template names displayed on 'Test result overview' page" => sub {
+    $driver->get('/group_overview/1002');
+    is($driver->find_element('.progress-bar-unfinished')->get_text(),
+        '1 unfinished', 'The number of unfinished jobs is right');
+
+    $driver->get('/tests/overview?distri=opensuse&version=13.1&build=0091&groupid=1002');
+    my @tds = $driver->find_elements('#results_DVD tbody tr .name');
+    is($tds[0]->get_text(), 'kde_variant', 'job template name kde_variant displayed correctly');
+
+    my @descriptions = $driver->find_elements('td.name a', 'css');
+    is(scalar @descriptions, 2, 'only test suites with description content are shown as links');
+    disable_bootstrap_animations;
+    $descriptions[0]->click();
+    is(wait_for_element(selector => '.popover-header')->get_text, 'kde_variant', 'description popover shows content');
+};
+
 subtest 'add comments' => sub {
-    my @buttons = $driver->find_elements('button[title="Add comments"]');
+    my @buttons = $driver->find_elements('button[title="Restart or comment jobs"]');
     is @buttons, 0, 'button for adding comments not present if not logged-in';
 
     $driver->find_element_by_link_text('Login')->click;
     $driver->get('/tests/overview?state=done&result=failed');
     disable_bootstrap_animations;
-    $driver->find_element('button[title="Add comments"]')->click;
+    $driver->find_element('button[title="Restart or comment jobs"]')->click;
     my $comment_text = 'comment via add-comments';
-    my $submit_button = $driver->find_element('#add-comments-controls button[type="submit"]');
-    $driver->find_element_by_id('text')->send_keys($comment_text);
-    is $submit_button->get_text, 'Submit comment on all 2 jobs', 'submit button displayed with number of jobs';
+    my $submit_button = $driver->find_element('#add-comments-controls button[id="commentJobsBtn"]');
+    $driver->find_element_by_id('text')->send_keys($comment_text);    #sleep(300);
```

Comment:
/
Don't add disabled code as nobody but you will understand why this is not active. Better leave it out and keep in your local git repository, either in `git stash` or a temporary "WIP"-commit

See https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/CONTRIBUTING.md#coding-style for more details